tests/foreman/cli/test_organization.py

```diff
@@ -55,26 +55,31 @@ def valid_labels_list():
         gen_string('alpha'),
         gen_string('alphanumeric'),
         gen_string('numeric'),
-        '{}-{}'.format(gen_string('alpha', 5), gen_string('alpha', 5)),
-        '{}_{}'.format(gen_string('alpha', 5), gen_string('alpha', 5)),
+        f"{gen_string('alpha', 5)}-{gen_string('alpha', 5)}",
+        f"{gen_string('alpha', 5)}-{gen_string('alpha', 5)}",
     ]
 
 
-class OrganizationTestCase(CLITestCase):
-    """Tests for Organizations via Hammer CLI"""
+@pytest.fixture
+def _make_proxy(request):
+    """Create a Proxy and clean up when done."""
+    options = getattr(request, 'param', None)
+    proxy = make_proxy(options=options)
+    yield proxy
+    capsule_cleanup(proxy['id'])
+
 
-    def _make_proxy(self, options=None):
-        """Create a Proxy and register the cleanup function"""
-        proxy = make_proxy(options=options)
-        # Add capsule to cleanup list
-        self.addCleanup(capsule_cleanup, proxy['id'])
-        return proxy
+@pytest.fixture
+def _make_org(request):
+    """Create an Organization and clean up when done."""
+    options = getattr(request, 'param', None)
+    org = make_org(options=options)
+    yield org
+    org_cleanup(org['id'])
 
-    @classmethod
-    def setUpClass(cls):
-        """Create an organization."""
-        super().setUpClass()
-        cls.org = make_org()
+
+class TestOrganization:
```

Comment:
any reason to keep the class?

Reply:
Any reason to remove it? More work to remove the class for no benefit that I see, but if we're going to make it a rule to remove classes, I can do it.

Reply:
Done.